### DISCUSSION TITLE
Restaure Repositories

### DISCUSSION THREAD

**Discussion by gabrielschiavis** [community/community#59607] [New to GitHub] [answered]
So, I've tried restoring a repository and it didn't appear on my repositories, any other repository I'd restaure have shown immediately on my page. I deleted just now by accident. Please someone help hahahahahaha I'm in a crises!

**Comment by queenofcorgis** [comment, marked as answer]
It can take up to an hour after a repository is deleted before that repository is available for restoration. Check out the documentation we have for more details [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#about-repository-restoration).
